<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="language_label">
         <property name="font">
          <font>
           <family>Aparajita</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>語言:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <widget class="QCheckBox" name="c_checkBox">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>C</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cpp_checkBox">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cpp</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Project title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title_lineEdit">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="path_label">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Folder to create project in:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QLineEdit" name="path_lineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="path_pushButton">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="create_pbtn">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
